### DISCUSSION TITLE
Autofill user details on login

### DISCUSSION THREAD

**Discussion by mohitkumar29** [swellstores/community#517] [Help] [answered]
Hey there,

having an issue on the swell hosted checkout page, 

1. the name fields are still there even when the user is logged in? (screenshot attached)
2. after i skipped past that shouldn't the billing address fields be auto filled since the user is already logged in and there is an address stored in the account?

![image](https://github.com/swellstores/community/assets/104753963/756af43c-b18d-4d2c-bbe8-6a59ea278c5f)
![image](https://github.com/swellstores/community/assets/104753963/bed16d65-f532-425b-a858-57160e0bddfe)

**Comment by logeshswell** [comment, marked as answer]
> the name fields are still there even when the user is logged in?

I can confirm that it's a frontend bug. Let me take a deeper look at this issue and get back to you if I get some updates.

**Reply by logeshswell** [reply]
The checkout is expected to redirect to Payment page but it's redirecting to Shipping page which is not applicable for digital product.

**Reply by logeshswell** [reply]
This issue is fixed at our end. Can you check and confirm?

**Comment by logeshswell** [comment]
> shouldn't the billing address fields be auto-filled?

It will be auto-filled if the customer contains at least one card with the associated billing details.
